packages/dds/tree/api-report/tree.public.api.md

```diff
@@ -514,10 +525,11 @@ export interface TreeNodeSchemaCore<out Name extends string, out Kind extends No
     readonly info: Info;
     // (undocumented)
     readonly kind: Kind;
+    readonly metadata?: RecursiveReadonly<TMetadata> | undefined;
```

Comment:
I think it would be better to structure this like `readonly metadata: NodeSchemaMetadata<TMetadata>;

This prevents people from smuggling in top level fields outside of custom in the type. As this type is sealed, we can add required fields just fine

Reply:
Yeah, had this change prepped but not pushed. Now pushed.